src/java.base/share/classes/jdk/internal/util/HexDigits.java

```diff
@@ -114,24 +114,18 @@ public static short digitPair(int i, boolean ucase) {
     }
 
     /**
-     * Return a little-endian packed integer for the 4 ASCII bytes for an input unsigned 2-byte integer.
-     * {@code b0} is the most significant byte and {@code b1} is the least significant byte.
-     * The integer is passed byte-wise to allow reordering of execution.
+     * Insert the int into the buffer as 4 hexadecimal digits
```

Comment:
4 hex digits is 2 bytes, right? 
should this take a short instead?
At a minimum, seems documentation should be clear as to which 2 bytes are used.